examples/using-reach-skip-nav/README.md

```diff
@@ -0,0 +1,32 @@
+# Using @reach/skip-nav
```

Comment:
I still feel like this example needs to describe what it does, and the hook into client-side routing is significant to mention in addition to the "skip-nav" part.